### PR TITLE
Add a template for bug report and feature request in Github repo

### PR DESCRIPTION
**Pull request by laomuon**
This PR is to add a template when creating a bug report or a feature request to the Github repo, so that the one who create it can provide all the informations needed

### CODE REVIEW

**Comment by laomuon** [comment]
@sebastinas I think this can be a positive thing to the development of the repo without any modification to the codebase

**Comment by sebastinas** [comment]
Can this be implemented on a organization level instead?

File: .github/ISSUE_TEMPLATE/config.yml [hunk 1]
```diff
@@ -0,0 +1 @@
+blank_issues_enabled: false
```
Comment (alerque) [inline .github/ISSUE_TEMPLATE/config.yml:1]: Please don't do this. I understand why templates help most people remember what they should include in a good bug report, but disabling this means people like me can't use extensive history as FOSS maintainers to know what we should write and use CLI tools to manage bugs.

**Comment by laomuon** [comment]
> Can this be implemented on a organization level instead?

I think it will be better if every repo has its own template

**Comment by sebastinas** [comment]
> > Can this be implemented on a organization level instead?
> 
> I think it will be better if every repo has its own template

Why though? It's essentially the same for all the repos.

**Comment by sebastinas** [comment]
This has been implemented on the organization level.

**Pull request closed without merging**